src/builder/modules/Generator.res

```diff
@@ -199,6 +316,34 @@ let drawTextureLegacy = (
   )
 }
 
+let drawCuboid = (
```

Comment:
This is a great helper function.

I'm thinking that this one actually belongs in a new module rather than in the builder, since it's specific to Minecraft textures (specifically how it orients it's texture patterns).

I am thinking we create a new module next to the `TextureMap.res` module and name it `Minecraft.res` and let's move this function over there.

What do you think?


I am thinking we will move some of the code in `TextureMap.res` into this new module as well, but that change is for a separate PR.

Reply:
That does make sense to do, but what would be in `TextureMap.res` and what would be in `Minecraft.res`?

Reply:
I suspect that probably everything will move across into `Minecraft.res` - I probably should have named it that from the beginning.